content/en/data_streams/messages.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Messages
+---
+
+The Messages feature allows you to view messages at specific offsets within Kafka. Inspecting these messages can be crucial when troubleshooting issues related to specific services or data streams.
+
+## Setup
+Before diving into detailed configuration steps, here's a quick overview of what's involved in setting up the Messages feature:
+
+| Step               | Description                                                                                                                                       | Setup Guides              |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|
+| Agent setup        | An agent needs Kafka consumer integration enabled, be on at least version 7.69.0, and have remote configuration enabled                           | [Agent Setup][1]          |
+| Verify permissions | You'll need a few log access permissions which usually come standard as well as `Data Streams Monitoring Capture Messages` enabled in Datadog     | [Required Permissions][2] |
+
+
+### Agent Setup
+
+#### Selecting an Agent
+Any Datadog agent having access to Kafka can be used for setup.
+
+If you self-host Kafka, set up Messages on your Kafka broker's agent. Otherwise, choose any agent your producer or consumer services communicate with that has access to your Kafka cluster.
+
+#### Step-by-step guide
+
+Complete the following steps on the same Datadog agent:
+
+##### 1. Verify agent version
+
+Ensure your agent is running version `7.69.0` or later.
+
+##### 2. Enable [remote configuration][3]
+
+Confirm Remote Configuration is enabled for your agent (typically enabled by default). Verify this setting on the [Fleet Automation page][4] and also confirm it at the organizational level via the [Remote Configuration settings page][5].
+
+##### 3. Configure Kafka consumer integration
+Create or update the Kafka consumer configuration file at
+
+```
+[agent config directory]/kafka_consumer.d/conf.yaml
```

Comment:
What do you think about updating the docs of the `kafka_consumer` integration, with this text, and linking to it from this doc?